doc/how_to/load_your_data_into_sorting.rst

```diff
@@ -0,0 +1,152 @@
+Load Your Own Data into a Sorting
+=================================
+
+Why make a :code:`Sorting`?
+
+SpikeInterface contains pre-build readers for the output of many common sorters.
+However, what if you have sorting output that is not in a standard format (e.g.
+old csv file)? If this is the case you can make your own Sorting object to load
+your data into SpikeInterface. This means you can still easily apply various
+downstream analyses to your results (e.g. building correlograms or for generating
+a :code:`SortingAnalyzer``).
+
+The Sorting object is a core object within SpikeInterface that acts as a convenient
+way to interface with sorting results, no matter which sorter was used to generate
+them. **At a fundamental level it is a series of spike times and a series of labels
+for each spike along with some associated metadata.** Below, we will show you have
+to take your existing data and load it as a SpikeInterface :code:`Sorting`` object.
+
+
+Reading a standard spike sorting format into a :code:`Sorting`
+-------------------------------------------------------------
+
+For most spike sorting output formats the :code:`Sorting` is automatically generated. For example one could do
+
+.. code-block:: python
+
+    from spikeinterface.extractors import read_phy
+
+    # For kilosort/phy output files we can use the read_phy
+    # most formats will have a read_xx that can used.
+    phy_sorting = read_phy('path/to/folder')
+
+And voila you now have your :code:`Sorting` object generated and can use it for further analysis. For all the
```

Comment:
```suggestion
And voilà you now have your :code:`Sorting` object generated and can use it for further analysis. For all the
```
Dude : P

Reply:
I can't add the accents on my Windows keyboard..... but I appreciate your accuracy.

Reply:
Just kidding here : )